pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-oai-pmh</artifactId>
-  <version>1.2.0-SNAPSHOT</version>
```

Comment:
No need to make this now. Version changing will happen during release plugin execution.